handler/handler.go

```diff
@@ -783,6 +783,7 @@ func (h *Handler) Open(ctx context.Context, params ...string) error {
 	// force error/check connection
 	if err == nil {
 		if err = drivers.Ping(ctx, h.u, h.db); err == nil {
+			h.l.Completer(drivers.NewCompleter(ctx, h.u, h.db, readerOpts(), completer.WithConnStrings(h.connStrings())))
```

Comment:
Add a check for `h.l.Interactive()` before calling this.

Reply:
Done in https://github.com/xo/usql/pull/481/commits/c0940a3a766548f81f4915edf9d6284a867d8e1a.